src/@next/Tag/Tag.tsx

```diff
@@ -51,7 +65,16 @@ export const Tag = React.forwardRef<HTMLDivElement, TagProps>(function Tag(
   );
 
   return (
-    <TagStyle ref={ref} {...props} value={value}>
+    <TagStyle
+      ref={ref}
+      {...props}
+      value={value}
+      onClick={!disabled && onClick}
+      data-clickable={!!onClick}
+      role={!!onClick ? 'button' : undefined}
+      data-disabled={!!onClick && disabled}
```

Comment:
why do we need to check onClick too for disabled here?

Reply:
Hmm, yes, I should delete the onClick. I was thinking of someone passing `disabled` in non clickable tag. But I think that would not make any sense.

Reply:
Fixed